embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/startup/AppStartupTraceEmitter.kt

```diff
@@ -376,6 +385,14 @@ internal class AppStartupTraceEmitter(
         startupActivityPostCreatedMs?.let { timeMs ->
             addAttribute("startup-activity-post-created-ms", timeMs.toString())
         }
+
+        sdkInitEndedInForeground?.let { inForeground ->
+            addAttribute("embrace-init-in-foreground", inForeground.toString())
```

Comment:
Should these be `emb` instead?

Reply:
I actually removed the `emb-` prefix from the other ones too. The reason for that is because a) this is just a custom attribute on a perf trace logged by us, not an attribute in a fixed schema; b) because it's logged by us, we won't have to worry about disambiguating this with customer data; and c) I'm not convinced that these attributes will be needed in the semantic conventions even if we make app startup spans a thing, so I just want to make them more one-off.

That said, this is mostly in my mind - I should validate this with the entire group first, but lets keep this for now.